internal/controller/stages/stages.go

```diff
@@ -70,12 +70,12 @@ type reconciler struct {
 	startVerificationFn func(
 		context.Context,
 		*kargoapi.Stage,
-	) (*kargoapi.VerificationInfo, error)
+	) *kargoapi.VerificationInfo
```

Comment:
Function signature changed...

If something goes wrong, we report it through the VerificationInfo's Phase and Message fields. What we don't want is the error to present as a _Stage_ error.